utils/build-script-impl

```diff
@@ -1774,7 +1774,6 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DSWIFT_PATH_TO_CMARK_SOURCE:PATH="${CMARK_SOURCE_DIR}"
                     -DSWIFT_PATH_TO_CMARK_BUILD:PATH="$(build_directory ${host} cmark)"
                     -DSWIFT_PATH_TO_LIBDISPATCH_SOURCE:PATH="${LIBDISPATCH_SOURCE_DIR}"
-                    -DSWIFT_PATH_TO_LIBDISPATCH_BUILD:PATH="$(build_directory ${host} libdispatch)"
```

Comment:
Are we sure this isn't used? Looks like the `lit` test suite uses this variable.

Reply:
So it is.  Good catch.  I think that we should compute that value instead.  I'll make some additional changes for this.